kafka-rest/src/main/java/io/confluent/kafkarest/resources/v3/AclsResource.java

```diff
@@ -56,17 +58,87 @@
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 import javax.ws.rs.core.Response.Status;
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.common.config.ConfigDef.Importance;
+import org.apache.kafka.common.config.ConfigDef.Type;
 
 @Path("/v3/clusters/{clusterId}/acls")
 public final class AclsResource {
 
+  private static final String API_V3_ACLS_ENABLE_CONFIG = "api.v3.acls.enable";
```

Comment:
What if you took a single config that takes a list of resource apis like a feature lst. 

ap1.v3.enable = [CLUSTER, ACL, TOPIC]

And a second one for methods, although perhaps it would be better for this to be  negative list. 

api.v3.<resource_name>.disallow = [DELETE]